style/components/card.css

```diff
@@ -0,0 +1,98 @@
+img {
```

Comment:
can be moved to reset.css if you have one

Reply:
sure